kivy/core/text/markup.py

```diff
@@ -463,11 +463,11 @@ def render_lines(self, lines, options, render_text, y, size):
 
         for layout_line in lines:  # for plain label each line has only one str
             lw, lh = layout_line.w, layout_line.h
-            x = xpad
+            x = padding_right
```

Comment:
Should not be `padding_left` here, instead of `padding_right` ?

Reply:
This is interesting... I'm not sure if this code already expected to receive a value for the padding right (as previously the left and right values were the same), but passing the value of `padding_left` instead of `padding_right` will have the opposite effect of what is expected

Reply:
> will have the opposite effect of what is expected

I tested it this morning (before having the first coffee of the day), and changing `x = padding_right` to `x = padding_left`, actually worked with `halign = 'left'`.

What is your expectation, and how it looks instead?

Reply:
Oops, my bad, I think I got confused while testing the different alignment modes...

You're right! I tested again and it looks like changing `x = padding_right` to `x = padding_left` did work! Good catch!